### ISSUE TITLE
training parameters

### ISSUE THREAD

**Issue by ira-b-27** [dunbar12138/Audiovisual-Synthesis#23]
Hi,
Using what seemed to be default training parameters I was not able to train a model for JohnOliver_01.mp4 recording, on par with the one used in the examples on the project page. The results are suboptimal.
Can you tell what parameters you had used and on how many GPUs you trained in parallel? 

I trained with multigpu option, on 4GPUs, with a batch size of 32 (8 per GPU), 600 epochs which resulted in 30000 iterations. The learning rate lr=1e-3, as specified in Generator() definition in train_audio.py.

Are there any parameters I should change? 

Thanks in advance.
